### DISCUSSION TITLE
umi有关于monorepo的最佳实践相关的文档吗？

### DISCUSSION THREAD

**Discussion by xzp-git** [umijs/umi#12223] [Q&A]
![image](https://github.com/umijs/umi/assets/73266354/51ed927e-e37f-4152-9607-bbe0fbcebd87)
只看见介绍里面提了一下

**Comment by fz6m** [comment]
现在推荐使用 pnpm workspace 做 monorepo ，如果基建 ci 支持上传或保留上次的缓存，可以加上 turborepo 提高多子包的构建效率，并没有具体展开的 monorepo 实践介绍。

**Reply by zFitness** [reply]
> 现在推荐使用 pnpm workspace 做 monorepo ，如果基建 ci 支持上传或保留上次的缓存，可以加上 turborepo 提高多子包的构建效率，并没有具体展开的 monorepo 实践介绍。

哥们，现在这个项目蚂蚁内部有多少人维护？
